### PR TITLE
Globally increase tests timeouts

### PR DESCRIPTION
**Pull request by horia-delicoti**
[Fix #2 ]

### CODE REVIEW

File: src/mocha.opts [hunk 1]
```diff
@@ -1,3 +1,3 @@
 --require ./src/setup
 --ui bdd
---timeout 5000
+--timeout 900000
```
Comment (jviotti) [inline src/mocha.opts]: I think this is way too much (250 hours). Maybe something like 15 minutes should do the trick?

Comment (horia-delicoti) [reply src/mocha.opts]: As I understood, timeout it's in milliseconds. And 900000 is 15 minutes.

Comment (jviotti) [reply src/mocha.opts]: Ahhh, my bad :P Looks good then

Comment (horia-delicoti) [reply src/mocha.opts]: No worries :P

**Pull request merged**